src/scenes/home/informationForm/formComponents/interests.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Line } from 'rc-progress';
+import Form from 'shared/components/form/form';
+import FormCheckBox from 'shared/components/form/formCheckBox/formCheckBox';
+import PropTypes from 'prop-types';
+import styles from './formComponents.css';
+
+const languageValues = ['Javascript', 'Ruby', 'Java', 'Python', 'C#', 'C', 'Swift', '.NET', 'HTML / CSS'];
```

Comment:
Should these two consts be moved into `shared/constants` to follow the pattern of some of the other components?